### PR TITLE
Example for multiple input files with SAS

### PR DESCRIPTION
**Pull request by zhieejhia93**
Added a new example to demonstrate how to use WATTS to run a SAS simulation that uses multiple input files. Some notes:

1. The problem is the same as the "1App_SAS_SodiumLoop" example with the template broken into multiple files.
2. Minor modifications to `PluginSAS` to allow the plugin to accept extra input files.

### CODE REVIEW

File: src/watts/plugin_sas.py [hunk 1]
```diff
@@ -93,8 +95,9 @@ class PluginSAS(TemplatePlugin):
 
     def  __init__(self, template_file: str, show_stdout: bool = False,
                   show_stderr: bool = False,
-                  extra_inputs: Optional[List[str]] = None):
-        super().__init__(template_file, extra_inputs)
+                  extra_inputs: Optional[List[str]] = None,
+                  extra_template_inputs: Optional[List[PathLike]] = None):
+        super().__init__(template_file, extra_inputs, extra_template_inputs)
```
Comment (nstauff) [inline src/watts/plugin_sas.py:101]: @zhieejhia93 and @paulromano - those changes to SAS plugin need also to be pushed to all plugin (PyARC, MOOSE) that will benefit from this `extra_template_inputs` option - should this be done at a "higher level" (`plugin` class) or pushed to the other plugins?

File: doc/source/user/usage.rst [hunk 2]
```diff
@@ -251,6 +251,15 @@ Similar to the SAS executable, the utilities are also OS dependent. To execute
...
 
+WATTS also supports the "INCLUDE" directive of SAS that allows for multiple input
+files. Users simply need to add "INCLUDE extra_input_file_names" to the template
+and specify the names of the extra input files as a string to the
+"extra_template_inputs" argument when calling the :class:`~watts.PluginSAS` class::
+
+    sas_plugin = watts.PluginSAS('sas_template', show_stdout=True, extra_template_inputs=['extra_input_file_names'])
+
+An example has been provided to demonstrate this capbility.
+
```
Comment (nstauff) [inline doc/source/user/usage.rst:262]: `An example has been provided to demonstrate this capbility` - fix typo (capability) and provide link to this new example.

File: examples/MultiTemplates_SAS_SodiumLoop/sas_template [hunk 3]
```diff
@@ -0,0 +1,199 @@
+
```
Comment (nstauff) [inline examples/MultiTemplates_SAS_SodiumLoop/sas_template]: the name of the example folder should be changed for consistency purposes - `1App_SAS_MultiTemplate-SodiumLoop`

**Review by nstauff — CHANGES_REQUESTED** [review]

File: src/watts/plugin_sas.py (hunk 1 above)
Comment (zhieejhia93) [reply src/watts/plugin_sas.py:101]: These are already in the `TemplatePlugin` class. I think we just need to add them to the MOOSE and PyARC plugins.

Comment (nstauff) [reply src/watts/plugin_sas.py:101]: OK - putting a new issue to make `extra_template_inputs` available on other plugins.

**Review by nstauff — APPROVED** [review]

File: src/watts/plugin_sas.py (hunk 1 above)
Comment (zhieejhia93) [reply src/watts/plugin_sas.py:101]: Sure, I can handle that. Thanks.

**Pull request merged**